### DISCUSSION TITLE
Nested update for one to one relations PrismaClientValidationError

### DISCUSSION THREAD

**Discussion by slicegcp** [prisma/prisma#15578] [Q&A] [answered]
Hi,

I have the below schema and try to do the below nested update.

TS doesn't complain however when I try to do the actual update I get a PrismaClientValidationError.

What is interesting is that it worked a few day ago but now suddenly something happened.

```prisma
model User {
  id           String   @id @unique
  capital      Int
  posts        Post[]
}

model Post {
  id        Int       @id @default(autoincrement())
  author    User      @relation(fields: [authorId], references: [id], onDelete: Cascade)
  authorId  String
  order     Order?
}

model Order {
  id           Int         @id @default(autoincrement())
  post         Post        @relation(fields: [postId], references: [id], onDelete: Cascade)
  postId       Int         @unique
  capital      Int
}
```
```ts
prisma.order.update({
    where: {
      id,
    },
    data: {
      post: {
        update: {
          author: {
            update: {
              capital: {
                increment: someValue,
              },
            },
          },
        },
      },
    },
  });
```
The error:
```
Unknown arg `post` in data.post for type OrderUncheckedUpdateInput. Did you mean `postId`? Available args:
```
Any thoughts?

Thank you,
Serban

**Comment by slicegcp** [comment, marked as answer]
Actually we get the wrong error from prisma client. The real error was that I was trying to increment with a decimal value (instead on Int)

**Reply by austincrim** [reply]
Were you able to resolve the issue? Feel free to open a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with a reproduction for the incorrect error message.

**Reply by slicegcp** [reply]
Yes, thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
